mito-ai/mito_ai/completions/prompt_builders/utils.py

```diff
@@ -0,0 +1,19 @@
+from typing import List, Optional
+from mito_ai.rules.utils import get_rule
+
+def get_rules_str(selected_rules: Optional[List[str]]) -> str:
+    """
+    Get a string of the rules that the user has selected.
+    """
+    if selected_rules is None:
+        return ''
+    
+    rules_str = ''
+    for rule in selected_rules:
+        rule_content = get_rule(rule)
+        if rule_content is None or rule_content == '':
+            continue
+        
+        rules_str += f"===========\n\nCustom Instructions Provided by User: {rule}\n\n{rule_content}\n\n==========="
```

Comment:
Would it be more useful to add stronger wording here such that it does not get shrouded amidst all the other context ? 


```suggestion
        rules_str += f"===========\n\ESSENTIAL BEHAVIORAL MODIFICATION:
Integrate and enforce this user-defined rule as a core operational parameter: {rule}
\n\n{rule_content}\n\n==========="
```

Reply:
I'd be curious to see how well this works (evals anyone?). fwiw, just based on past prompting experience, the simplest explanations usually work best.